src/main/scala/org/constellation/domain/transaction/PendingTransactionsMemPool.scala

```diff
@@ -15,10 +15,20 @@ class PendingTransactionsMemPool[F[_]: Concurrent](semaphore: Semaphore[F])
       if (txs.size < 1) {
```

Comment:
Could be `if (txs.isEmpty) {`. Still cleaner than `== 0` and definitely cleaner than `< 1` as Map can't have negative size.